### DISCUSSION TITLE
[HELP] Qué nivel de HTML soportan los messages en webclient?

### DISCUSSION THREAD

**Discussion by Yiakman** [Cotalker/documentation#161] [2. Questions and Answers] [answered]
Estoy en la disyuntiva de enviar información mediante un formulario simple sin triggers o bien un mensaje con html para darle órden y formato. Cuál sería la jerarquía que impera para dar formato al mensaje en el HTML? Entiendo que no sirve de mucho añadir style como html-tag y cosas de ese estilo.
Sé que puedo usar <-table> <-strong> <-italic> <-ul> <-ol>, <-h1><-h2><-h3> y <-h4>. Hay algún feature poderoso 🥑 que me sirva o será estéticamente preferente usar formularios?

**Comment by Yiakman** [comment]
Acabo de probar <p- style="color:#FF0000";>Red paragraph text<- /p>
Funciona

**Comment by rafaelwalter** [comment, marked as answer]
@Yiakman te refieres al message+html o message+plain?

**Reply by rafaelwalter** [reply]
message+html corre en un iframe y puede correr una app completa en html y javascript. 
Los message+plain no tienen soporte oficial para html, no se debe utilizar html ya que se poderia quitar en cualquier nuevo release.

**Reply by Yiakman** [reply]
uso text/system. Hay una configuración aparte para colocar message+html?

**Reply by rafaelwalter** [reply]
text/system tampoco tienen soporte oficial para html. Solamente para markdown.

Para ocupar message+html debes configurar un COTMessage como:
````
{
  contentType:'text/html',
  contentUrl: '', // URL para content (optional)
  content: '' // inline html (optional)
}
````

**Reply by Yiakman** [reply]
Buena Rafa, sabes de alguna limitación respecto al ancho que va a resultar del iframe? Pregunto para pensar el html acorde a eso

**Reply by rafaelwalter** [reply]
El ancho del iframe depende del ancho del chat. Los mensajes tienen un maximo de aproximadamente 80% del ancho total del chat.
